src/types.ts

```diff
@@ -635,6 +635,7 @@ export interface ManifestOptions {
    */
   scope_extensions: {
     origin: string
+    type?: 'origin'
```

Comment:
add a type: `export type ScopeExtensionsType = 'origin'` then use `type?: StringLiteralUnion<ScopeExtensionsType> | ScopeExtensionsType`, this way we can add future values for the type and TS won't complain: `export type ScopeExtensionsType = 'origin' | '<other1>' | ....` without the need to explicitly set the values, rn if a new type is added you need to use `unknown as any`  or similar hacks.

Reply:
`StringLiteralUnion` should be there in the `src/types.ts` module

Reply:
add also a `@see https://developer.mozilla.org/en-US/docs/Web/Progressive_web_apps/Manifest/Reference/scope_extensions` to the jsdoc

Reply:
add also a jsdoc to the type with `@default 'origin'`